Demos/LightboxDemo/LightboxDemo/AppDelegate.swift

```diff
@@ -8,14 +9,13 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     return window
     }()
 
-  lazy var viewController: UIViewController = {
-    let controller = ViewController()
-    return controller
-    }()
-
   func application(application: UIApplication, didFinishLaunchingWithOptions launchOptions: [NSObject: AnyObject]?) -> Bool {
 
-    window!.rootViewController = viewController
+    let imageNames = ["photo1", "photo2", "photo3"]
+    let images = imageNames.map { UIImage(named: $0)! }
```

Comment:
@vadymmarkov there is something delicious about `map`. I know that it sounds weird but it's just so cozy.